apps/worker/src/request_processor/request.processor.service.ts

```diff
@@ -49,7 +49,7 @@ export class RequestProcessorService extends BaseConsumer {
       // exponential backoff
       const delayedTime = 2 ** data.dependencyAttempt * this.configService.assetUploadVerificationDelaySeconds * MILLISECONDS_PER_SECOND;
       await job.moveToDelayed(Date.now() + delayedTime, job.token);
-      await job.update(data);
+      await job.updateData(data);
```

Comment:
This change surprised me.

Reply:
Yep this function is not there in this new redis replacement, maybe dev changed the name to be more appropriate